source/brailleDisplayDrivers/handyTech.py

```diff
@@ -784,7 +802,7 @@ def __init__(self, model, keys, isBrailleInput=False):
 				self.dots = self._calculateDots()
 				if key in KEY_SPACES or (key in (KEY_LEFT, KEY_RIGHT) and isinstance(model,EasyBraille)):
 					self.space = True
-			if key >= KEY_ROUTING:
+			if key >= KEY_ROUTING and key < KEY_ROUTING + model.numCells:

```

Comment:
Quite trivial, but I belief this can be written as:

```
if KEY_ROUTING <= key < KEY_ROUTING+model.numCells:
```

Reply:
Didn't know that but looked it up and learned something. Thx. Seems my C background is showing.